tests/test_api.rs

```diff
@@ -4651,6 +4651,17 @@ fn allow_atomics_wait() {
   }
 }
 
+#[test]
+fn date_time_configuration_change_notification() {
+  let _setup_guard = setup::parallel_test();
+  let isolate = &mut v8::Isolate::new(Default::default());
+  isolate
+    .date_time_configuration_change_notification(v8::TimeZoneDetection::Skip);
```

Comment:
just calling the method here, don't know how to test this without mocking out timezone